### PR TITLE
Add "Onscreen UI (GX Touch & Ekrano)" option to display settings

### PR DESCRIPTION
**Pull request by blammit**

### CODE REVIEW

**Comment by mpvader** [comment]
Was it tested on a GX and does it work? Also from WASM ?

**Comment by chriadam** [comment]
I have tested it on both CerboGX and Wasm.  It works in both, however in wasm it takes some time for QtMqtt to inform us that the connection to the MQTT server is lost, so it just sits in an idle state for a couple of minutes until QtMqtt deigns to reconnect.

File: pages/DialogLayer.qml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+				//% "Rebooting..."
+				? qsTrId("dialoglayer_rebooting")
+				//% "Device has been rebooted."
+				: qsTrId("dialoglayer_rebooted")
+
+			// On device, dialog cannot be dismissed; just wait until device is rebooted.
+			dialogDoneOptions: VenusOS.ModalDialog_DoneOptions_OkOnly
+			footer.enabled: BackendConnection.type !== BackendConnection.DBusSource
+			footer.opacity: footer.enabled ? 1 : 0
+			closePolicy: C.Popup.NoAutoClose
```
Comment (chriadam) [inline pages/DialogLayer.qml:38]: should be Templates not Controls, I will update.

**Review by chriadam — APPROVED** [review]
LGTM.

**Pull request merged**